### PR TITLE
[WIP] Tidy up filter bar CSS

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

File: frontend/public/style/_common.scss [hunk 1]
```diff
@@ -82,10 +84,6 @@ $co-external-link-padding-right: 15px;
   }
 }
 
-.co-m-pane__filter-bar-group--full-width {
```
Comment (rhamilto) [inline frontend/public/style/_common.scss:85]: This rule is orphaned.

File: frontend/public/style/_common.scss [hunk 2]
```diff
@@ -48,20 +64,6 @@ $co-external-link-padding-right: 15px;
...
 }
 
-.co-m-pane__filter-bar,
-.co-m-pane__help-text {
-  margin: 20px ($grid-gutter-width / 2);
-  @media (min-width: $grid-float-breakpoint) {
-    margin-left: $grid-gutter-width;
-    margin-right: $grid-gutter-width;
-    margin-top: $grid-gutter-width;
-    &--alt {
```
Comment (rhamilto) [inline frontend/public/style/_common.scss:58]: This nesting is resulting in an orphaned `.co-m-pane__help-text--alt` rule.

**Comment by bipuladh** [comment]
> @bipuladh, we may want to simply integrate these changes in to your forthcoming PR, but I wanted to document them here so as not to lose track of them in the shuffle.

@rhamilto thank you for documenting them. I will add them in my PR.

**Comment by rhamilto** [comment]
Included in #5227

**Pull request closed without merging**